openvdb_points/tools/PointDataGrid.h

```diff
@@ -328,6 +344,32 @@ class PointDataLeafNode : public tree::LeafNode<T, Log2Dim> {
         mAttributeSet->appendAttribute(attribute, expected, replacement);
     }
 
+    AttributeArray& attributeArray(const size_t pos)
```

Comment:
I still don't really see any reason to expose this at the moment with the exception of the sorting routine, so I still feel like there might be a better way of achieving this. I can take a look at the changes you've made to the sort tomorrow and see if we can find another way.